xamarin/Xamarin.Forms
# Fix 9123 - [UWP] Use style to show BackgroundColor on Button / ImageButton when hovering

### Description of Change ###

By applying a style (a copy of the default Button style in UWP version 16399) instead of relying on the default Button style, this PR ensures that setting BackgroundColor for a Button or an ImageButton is respected when the mouse pointer is hovered above the Button / ImageButton. Without this the Button or ImageButton BackgroundColor turns grey when hovering the mouse pointer on UWP TargetPlatformVersion 17763 or above, due to changes in the default Button style's ControlTemplate.
 
### Issues Resolved ### 

- fixes #9123 

### API Changes ###
 
 None

### Platforms Affected ### 

- UWP

### Behavioral/Visual Changes ###
On UWP 17763 and above, BackgroundColor for Button and ImageButton wil be respected when hovering mouse pointer, instead of turning grey.

### Before/After Screenshots ### 
Screenshots from UWP TargetPlatformVersion 18362:

Before fix, no hover:
![fix-9123-before-no-hover](https://user-images.githubusercontent.com/3807144/73024235-7a07bf80-3e2d-11ea-81e9-cb13f076b000.png)

Before fix, hover over Button:
![fix-9123-before-hover-button](https://user-images.githubusercontent.com/3807144/73024297-97d52480-3e2d-11ea-87d0-57607abb82f1.png)

Before fix, hover over ImageButton:
![fix-9123-before-hover-image-button](https://user-images.githubusercontent.com/3807144/73024321-a4597d00-3e2d-11ea-8144-8c60c9c2456b.png)

After fix, no hover:
![fix-9123-after-no-hover](https://user-images.githubusercontent.com/3807144/73024352-b76c4d00-3e2d-11ea-84b2-33d92819d483.png)

After fix, hover over Button:
![fix-9123-after-hover-button](https://user-images.githubusercontent.com/3807144/73024380-c3580f00-3e2d-11ea-8305-6a2c8fa1bf69.png)

After fix, hover over ImageButton:
![fix-9123-after-hover-image-button](https://user-images.githubusercontent.com/3807144/73024404-d10d9480-3e2d-11ea-8467-72dc74fed946.png)


### Testing Procedure ###
Be sure to compile and run the ControlGallery app with UWP TargetPlatformVersion 17763 or above. Go to Test Case 9123. Hover the mouse pointer over the Button and the ImageButton, and make sure the BackgroundColor remains yellow / yellow-green.

### PR Checklist ###
